pkg/stack/template.go

```diff
@@ -360,6 +361,9 @@ func VersionsSince(repoURL string, sinceString string) ([]string, error) {
 		return []string{}, err
 	}
 
+	// After this, the versions are properly sorted
+	sort.Strings(tagsSince)
```

Comment:
This does a string based sort.

Chances are that version 0.9.0 comes *after* 0.10.0 because 1 comes earlier in the alphabet than 9.

Please do a sort with a custom sort function: `since.Compare(tag)` could work as one.

https://gobyexample.com/sorting-by-functions